latency/src/bin/z_ping.rs

```diff
@@ -80,6 +84,13 @@ async fn parallel(opt: Opt, config: Config) {
     let mut key_expr_ping = 0;
     if opt.use_expr {
         key_expr_ping = session.declare_expr("/test/ping").await.unwrap();
+        if opt.declare_publication {
+            session.declare_publication(key_expr_ping).await.unwrap();
+        }
+    } else {
+        if opt.declare_publication {
+            session.declare_publication("/test/ping").await.unwrap();
```

Comment:
Consider to define and use a string literal since "/test/ping" appears more than once in this program, for maintainability.

Reply:
Edited.